SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -156,6 +156,11 @@
       <Output TaskParameter="IsTest" PropertyName="SonarQubeTestProject" />
     </IsTestFileByName>
 
+    <PropertyGroup>
+        <!-- This can't be set in the FX-Cop property group as this target needs to run first -->
+        <SonarQubeRunMSCodeAnalysis Condition="$(SonarQubeTestProject) == 'true' ">false</SonarQubeRunMSCodeAnalysis>
```

Comment:
I've moved this assignment to "OverrideCodeAnalysisProperties" so the only responsibility for this target is to carry out the categorisation of the project type.